src/main/resources/messages.properties

```diff
@@ -74,4 +74,9 @@ validation.date.invalid.approval.date = The date cannot be on or before the end
 
 validation.characters.invalid.accounting_policies.basis_of_measurement_and_preparation = Enter valid characters
 validation.element.missing.accounting_policies.basis_of_measurement_and_preparation = Give details of how your accounts were prepared
-validation.length.maxExceeded.accounting_policies.basis_of_measurement_and_preparation = You must not exceed {0} characters
\ No newline at end of file
+validation.length.maxExceeded.accounting_policies.basis_of_measurement_and_preparation = You must not exceed {0} characters
+validation.length.maxExceeded.accounting_policies.basis_of_measurement_and_preparation = Basis of preparation must not exceed {0} characters
```

Comment:
I assume this is a merge issue, but this string can be removed because the one above replaced it

Reply:
it was.